ui/pages/home/index.scss

```diff
@@ -176,4 +176,35 @@
       margin-left: 44px;
     }
   }
+
+  &__new-tokens-imported-notification {
+    margin-bottom: 4px;
+    margin-inline-start: 8px;
+    padding: 8px;
+
+    @media screen and (min-width: $break-large) {
+      margin-bottom: 50px;
+      margin-inline-end: 8px;
+      left: 70%;
```

Comment:
Should this be on the left for RTL?

Reply:
@darkwing I am facing this issue when I remove the left alignment here, can you help me tackle this?

https://user-images.githubusercontent.com/43930900/164549440-bfacb624-b5c5-47da-88a7-3585f071e5b7.mov